kubeadm/scripts/PrepareNode.ps1

```diff
@@ -100,7 +100,9 @@ $cmd_commands=@("C:\k\kubelet.exe ", '$global:KubeletArgs ', '--cert-dir=$env:SY
 if ($CurrentVersion -lt $V1_24_Version) {
     $cmd_commands = $cmd_commands + "--network-plugin=cni " + "--image-pull-progress-deadline=20m "
 }
-$StartKubeletFileContent = '$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
+$StartKubeletFileContent = 'Import-Module "c:\k\hns.psm1"
+New-HnsNetwork -Type NAT -Name nat
```

Comment:
we shouldn't need to create the NAT network with Hostprocess approach.

Reply:
Bumped into an issue and from what I read, this was a solution to it. I will modify it back